docs/containers/kubernetes-env-vars.md

```diff
@@ -0,0 +1,96 @@
+---
+Area: containers
+ContentId: 8c1ee589-25bb-4c2d-ab8d-7ccfa7b4e282
+PageTitle: Use Kubernetes service environment variables for service to service communication
+DateApproved: 02/12/2021
+MetaDescription: Learn how to use Kubernetes service environment variables with Bridge to Kubernetes to enable service-to-service communication as a non-elevated user in a Kubernetes cluster
+---
+# Use Kubernetes service environment variables
```

Comment:
You might get a title word wrap. We can see once this is on the staging server.
The title could just be "Kubernetes service environment variables"
To avoid duplicate headers, you would need to change the H2 below, maybe just "Service environment variables"